synapse/rest/client/v2_alpha/account.py

```diff
@@ -114,6 +115,9 @@ async def on_POST(self, request):
             if self.config.request_token_inhibit_3pid_errors:
                 # Make the client think the operation succeeded. See the rationale in the
                 # comments for request_token_inhibit_3pid_errors.
+                # Also wait for some random amount of time between 100ms and 1s to make it
+                # look like we did something.
+                await self.hs.clock.sleep(random.randint(1, 10) / 10)
```

Comment:
Should we do this on `/submit_token` as well?

Reply:
Actually probably not as you're still making database calls, which will take time.